### PR TITLE
Adding GitHub username to table

### PR DESCRIPTION
**Pull request by gvicas17**
### Description
  Add description here
Added github username to user table
Added github username to user object in routes/user.js
Added username endpoint in routes/user.js

Allows storage of username to table for github calendar

### Tests
  How was it tested?
Tested on local server
### Reviewers
  2 reviewers required to merge to main. Must not merge your own pull request.

### CODE REVIEW

File: routes/user.js [hunk 1]
```diff
@@ -95,6 +96,19 @@ app.put('/displayname', verifyUser, (request, response) => {
...
 
+//Update user's GitHub username
+app.put('/githubusername', verifyUser, (request, response) => {
+  const { userID, gitHubUsername } = request.body;
+  User.update(userID, { github_username: gitHubUsername })
+    .then(() => {
+      response.status(200).json({ message: 'Updated user\'s github username successfully' });
+    })
+    .catch((err) => {
+      console.log(err);
```
Comment (robo-chick) [inline routes/user.js:107]: Don't forget to remove console.log :)

**Review by robo-chick — APPROVED** [review]
See note about console.log, otherwise awesome work!

**Review by ialkamal — APPROVED** [review]
Approved

**Review by Plofland — APPROVED** [review]
Great job.

**Pull request merged**